packages/cxl-ui/src/components/cxl-save-playbooks.js

```diff
@@ -0,0 +1,94 @@
+import '@conversionxl/cxl-lumo-styles';
+import { customElement, html, LitElement, property, query } from 'lit-element';
+import cxlSavePlaybooksStyles from '../styles/cxl-save-playbooks-css';
+
+@customElement('cxl-save-playbooks')
+export class CXLSavePlaybooksElement extends LitElement {
+  @query('a')
+  anchor;
+
+  @query('a > iron-icon')
+  ironIcon;
+
+  /**
+   * Is playbook currently in the "saved" state
+   */
+  @property({ type: Boolean, reflect: true })
+  selected;
+
+  /**
+   * if true, text "Save / Unsave Playbook" won't be displayed
+   */
+  @property({ type: Boolean, reflect: true })
+  starOnly;
+
+  @property({ type: Number, reflect: true })
+  playbookId;
+
+  @property({ type: Number, reflect: true })
+  userId;
+
+  /**
+   * API Url to which we make a POST request
+   *
+   * @type {string}
+   */
+  @property({ type: String, reflect: true })
+  apiUrl = 'https://jsonplaceholder.typicode.com/posts';
```

Comment:
This does not need to be an observed property as it never changes.

```suggestion
  apiUrl = 'https://jsonplaceholder.typicode.com/posts';
```

Reply:
@anoblet what about the `tests` of the component that should be written in the future, how would we mock `apiUrl` in that case?

Reply:
If we wanted to pass this property in we could use the `<cxl-save-playbook .apiUrl></cxl-save-playbook>` syntax. The reason why this isn't observed is that it won't change in the lifecycle of the component.

Reply:
@anoblet in this case shouldn't we remove `reflect` from the following properties if that's the case?

```
  @property({ type: Boolean, reflect: true })
  isCardVersion;

  @property({ type: Number, reflect: true })
  playbookId;

  @property({ type: Number, reflect: true })
  userId;
```

because all of them should be passed only once - when HTML is loaded, and won't be changed after the `DOMContentLoaded` event

Reply:
@anoblet just checked now that
`<cxl-save-playbook .apiUrl="..."></cxl-save-playbook>` this does not work (does not pass the property value).
Tried variants:
- no property defined
- defined as follows
```
  @property({ type: String })
  apiUrl = 'https://jsonplaceholder.typicode.com/posts';
```

Reply:
Removing reflect when not needed is good practice.

Strange that the `.myProperty=""` syntax doesn't work. Let me take a look into it.

If you want to keep the property observed for now, that should be fine :)

Reply:
@dominik-stypula-polcode https://lit-and-friends.slack.com/archives/C03PF4L4L/p1623763844090700

Reply:
@anoblet sorry I can't acces their workspace: `dominik.stypula@polcode.net doesn’t have an account on this workspace.`, and there is no option to create one there...

Reply:
@anoblet ok, never mind, I've created it on the other page.